<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicensesDialog</class>
 <widget class="QDialog" name="LicensesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Active licenses</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="requestEdit"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="generateButton">
     <property name="text">
      <string>Generate request</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="applyEdit"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>Apply license</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="refreshButton">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
